main.adb:17:38: error: "Constraint_Error" would have been raised at run time
main.adb:17:38: error: error in inlined body at line 14
main.adb:17:38: error: value not in range of type "Word64_Pos" defined at line 9
gnatprove: error during generation of Global contracts
